### PR TITLE
Eregcsc 1374, Update sidebar based on what they click

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves #
1374 feedback
**Description-**

Hides "Show all subpart A resources link if an orphan section is selected.

Fixes the count of supplemental content in a subpart for "Show all subpart {X} resources" number.  Before number was incorrect.
**This pull request changes...**

- expected change 1

**Steps to manually verify this change...**

1. steps to view and verify change

### CODE REVIEW

File: solution/ui/regulations/js/api.js [hunk 1]
```diff
@@ -397,7 +397,12 @@ const v3GetSupplementalContent = async (apiURL, {locations, locationDetails=fals
 
 }
 
-// API Functions Insertion Point (do not change this text, it is being used by hygen cli)
+const getSubpartTOC = async (apiURL, title, part, subPart) => 
+{
+    const url = apiURL.replace('/v2/', '/v3/')
+
+    return httpApiGetLegacy(`${url}title/${title}/part/${part}/version/latest/subpart/${subPart}/toc`)
+}// API Functions Insertion Point (do not change this text, it is being used by hygen cli)
```
Comment (knollfear) [inline solution/ui/regulations/js/api.js:405]: Can we please get a new line added here?

**Review by knollfear — COMMENTED** [review]
I had an issue when I went to a subpart via a fragment link.  When I go to the subpart with a more normal path it works, but https:// . . . #A it doesn't work right.

File: solution/ui/regulations/js/api.js (hunk 1 above)
Comment (thwalker6) [reply solution/ui/regulations/js/api.js]: done

**Comment by thwalker6** [comment]
Fixed the #A not working.  The logic is that if it is not a number than will take the subpart and pull those resources for the subpart

**Review by knollfear — APPROVED** [review]
LGTM

**Pull request merged**